src/app/layers/symbology/symbology.model.ts

```diff
@@ -193,11 +199,11 @@ export class SimplePointSymbology extends AbstractVectorSymbology implements ISi
   }
 
   getOlStyle(): ol.style.Style {
-      return new ol.style.Style({
-          image: new ol.style.Circle({
+      return new OlStyleStyle({
```

Comment:
Again, is it necessary to use `OlStyleStyle`?

Reply:
yes